src/main/java/es/deusto/spq/server/data/dto/ReviewDTO.java

```diff
@@ -31,25 +31,15 @@
 	 * The date the review was published.
 	 */
 	private Timestamp publishDate;
-	/**
-	 * The hotel DTO where the review is posted.
-	 */
-	private HotelDTO hotel;
-	/**
-	 * The user DTO that posts the review.
-	 */
-	private UserDTO user;
 
 	/**
 	 *
 	 * @param reviewID    the id to identify the review if needed.
 	 * @param opinion     the opinion of the user about a hotel.
 	 * @param score       the score(0-10) that a user gives a hotel..
 	 * @param publishDate the publishDate of the review.
-	 * @param hotel       the hotel DTO from the hotel that is the review abou.t
-	 * @param user        the user DTO of the user that has posted the review.
 	 */
-	public ReviewDTO(String reviewID, String opinion, int score, Timestamp publishDate, HotelDTO hotel, UserDTO user) {
+	public ReviewDTO(String reviewID, String opinion, int score, Timestamp publishDate) {

```

Comment:
If we don't provide the Hotel and User DTOs, what about providing their ids?

Reply:
I remove it because it was producing erros and they are only necesary for getting them from the DB, but i will try to find a solution and add the ids at least.